### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by tbremer** [bendc/uilang#15]
I am getting ready to submit a PR, but was wondering what kind of support you wanted this script to have?

You currently don't support IE8, can that be assumed?

What about IE9? 10?

Thanks!

**Comment by andrejsm** [comment]
Element.classList is supported starting with IE 10. See MDN [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element.classList#Browser_compatibility). Otherwise shim is required.

**Comment by bendc** [comment]
Yup, IE10+ is required.

**Comment by patrickcate** [comment]
Browser support should be added to http://uilang.com/

**Comment by tbremer** [comment]
I agree with @patrickcate. Also, updating the script to move away from `classList` would be an effortless change if you want to broaden support to IE9. 

I can make a PR.

**Comment by ghost** [comment]
Submitted a pull request for this #17.
